src/__integration_tests__/checkout.integration.test.ts

```diff
@@ -146,4 +152,50 @@ describe("Checkout payment activation tests", () => {
 
     expect(resultMessage).toContain("Spiacenti, si è verificato un errore imprevisto");
   });
+
+  it("Should show up threshold disclaimer (why manage creditcard)", async () => {
+    /*
+     * Credit card manage psp
+    */
+    const resultMessage = await checkPspDisclaimerBeforeAuthorizePayment(
+      PSP_ABOVETHRESHOLD,
+      VALID_FISCAL_CODE,
+      EMAIL,
+      VALID_CARD_DATA
+    );
+
+    expect(resultMessage).toContain("Perché gestisce la tua carta");
+
+    await cancelPayment();
+  });
+
+  it("Should show below threshold disclaimer (why is cheaper)", async () => {
+    /*
+     * Cheaper psp
+    */
+    const resultMessage = await checkPspDisclaimerBeforeAuthorizePayment(
+      PSP_BELOWTHRESHOLD,
+      VALID_FISCAL_CODE,
+      EMAIL,
+      VALID_CARD_DATA
+    );
+
+    expect(resultMessage).toContain("Suggerito perché il più economico");
+
+    await cancelPayment();
```

Comment:
Can you also avoid calling this method ?

Reply:
Cannot avoid calling this method, since a listener prevent the browser closure. So, before to close the browser jest cancel the payment and then close the browser.